server/scripts/exportMapping.ts

```diff
@@ -0,0 +1,33 @@
+import { writeFileSync, existsSync, mkdirSync } from 'fs'
+import { Photon } from '@prisma/photon'
+import { exportMapping } from '../src/resolvers/mapping'
+
+const photon = new Photon()
+
+const main = async (outputDirectory: string) => {
+  if (!existsSync(outputDirectory)) {
+    mkdirSync(outputDirectory)
+  }
+
+  let sources = await photon.sources.findMany()
+  for (const source of sources) {
+    console.log('source:', source)
+    const content = await exportMapping(photon, source.id)
+    writeFileSync(`${outputDirectory}/${source.id}.json`, content)
```

Comment:
I would use ${source.template.name}_${source.name}.json as file name in order for it to be recognizable.

Reply:
(there can't be two sources with the same name for a single template)

Reply:
Why not, but does it need to be recognizable? as a general rule I prefer not to use dynamic entity names as filenames as they can contain any character.

Reply:
maybe not because we'll probably want to re-import all sources if we lose data but it might come in handy. Leave it like this for now, we'll see if we ever need it to be human-readable!